external/wakaama/examples/shared/Make.defs

```diff
@@ -38,5 +38,9 @@
 CSRCS += commandline.c connection.c platform.c memtrace.c
 CFLAGS+=-I$(TOPDIR)/../external/wakaama/core
 
+ifeq ($(CONFIG_LWM2M_SECURITY),y)
```

Comment:
Dear. dr-venkman

Sorry for late response.
We're preparing next pull request including separation of example for DS SW R&D center
So, this option will not be used on wakaama examples in external folder

Thanks